### PR TITLE
Update Ansible, add Liquibase logo.

### PR DESCRIPTION
**Pull request by jbennett-liquibase**

### CODE REVIEW

File: build_scripts_examples/Ansible_Tower/liquibase_playbook.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    LB_TMP_DIR: /tmp/Toolbox
+  tasks:
+###
+### Checkout repository
+### This example uses a personal access token, adjust as required
+###
+    - name: Checking out repository
+      ansible.builtin.git:
+        repo: "https://{{ GITHUB_TOKEN }}@github.com/jbennett-liquibase/Toolbox.git"
+        dest: "{{ LB_TMP_DIR }}"
```
Comment (molivasdat) [inline build_scripts_examples/Ansible_Tower/liquibase_playbook.yml:48]: Does dest: create a /tmp/Toolbox/Toolbox or does it do more of a git clone URL outputdir = LB_TMP_DIR?

**Review by molivasdat — COMMENTED** [review]
Left one comment otherwise LGTM

File: build_scripts_examples/Ansible_Tower/liquibase_playbook.yml (hunk 1 above)
Comment (jbennett-liquibase) [reply build_scripts_examples/Ansible_Tower/liquibase_playbook.yml:48]: It creates the LB_TMP_DIR directory and then does a clone into it.

**Review by molivasdat — APPROVED** [review]
LGTM

**Pull request merged**